src/lib/ledger_builder_controller/ledger_builder_controller.ml

```diff
@@ -123,45 +124,20 @@ end = struct
 
   let ledger_builder_io {ledger_builder_io; _} = ledger_builder_io
 
-  let load_tip_and_genesis_hash
-      (controller : (Tip.t, State_hash.t) With_hash.t Store.Controller.t)
-      {Config.longest_tip_location; genesis_tip; _} log =
-    let genesis_state_hash =
-      Protocol_state.hash genesis_tip.Tip.protocol_state
-    in
-    match%map Store.load controller longest_tip_location with
-    | Ok tip_and_genesis_hash -> tip_and_genesis_hash
-    | Error `No_exist ->
-        Logger.info log
-          "File for ledger builder controller tip does not exist. Using \
-           Genesis tip" ;
-        {data= genesis_tip; hash= genesis_state_hash}
-    | Error (`IO_error e) ->
-        Logger.error log
-          !"IO error: %s\nUsing Genesis tip"
-          (Error.to_string_hum e) ;
-        {data= genesis_tip; hash= genesis_state_hash}
-    | Error `Checksum_no_match ->
-        Logger.error log
-          !"Checksum from location %s does not match with data\n\
-            Using Genesis tip"
-          longest_tip_location ;
-        {data= genesis_tip; hash= genesis_state_hash}
+  let load_tip_and_genesis_hash {Config.genesis_tip; _} log =
+    let genesis_state_hash = Protocol_state.hash genesis_tip.state in
+    Logger.info log
```

Comment:
make an issue to fix this asap

Reply:
This is dead code now that we're not merging to master